docs/configuration/backends/kubernetes.md

```diff
@@ -51,6 +51,15 @@ See also [Kubernetes user guide](/user-guide/kubernetes).
 #
 # labelselector = "A and not B"
 
+# Value of `kubernetes.io/ingress.class` annotation that identify Ingress objects to be processed.
+# Note that Ingress objects with an empty or nonexistent `kubernetes.io/ingress.class` value
+# will also be processed.
+#
+# Optional
+# Default: traefik (process all Ingresses)
```

Comment:
I think we can make things easier by choosing the empty string as default. Specifically:

1. We don't specifically need to set `IngressClass` to `traefik` in tests.
1. We can simplify `shouldProcessIngress` (see below).